sdk/Sdk.Generators/FunctionsUtil.cs

```diff
@@ -53,5 +80,11 @@ internal static string GetFullyQualifiedMethodName(MethodDeclarationSyntax metho
 
             return $"{fullyQualifiedClassName}.{method.Identifier.ValueText}";
         }
+
+        internal static string GetFullyQualifiedMethodName(IMethodSymbol method)
```

Comment:
Should we update the `GetFullyQualifiedMethodName` method above to use this new one?

Reply:
Looks like there's no more references to the original one, so removed.